ubuntudesign/landscape.canonical.com
# Content feedback

## Done

Added new content changes to the try landscape page and features page.

## QA

Run ```npm install``` and ```gulp build``` to create the styles.

To QA the content changes go to [landscape gh pages](http://ubuntudesign.github.io/landscape.canonical.com/index.html) and view the features.html and try-landscape.html pages and make sure there are no spelling mistakes or content issues.
